src/main/java/org/spongepowered/mod/mixin/world/MixinWorld.java

```diff
@@ -288,4 +361,88 @@ public void setKeepSpawnLoaded(boolean keepLoaded) {
     public SpongeConfig getWorldConfig() {
         return this.worldConfig;
     }
+
+    @Override
+    public void playSound(SoundType sound, Vector3d position, double volume) {
+        this.playSound(sound, position, volume, 1);
+    }
+
+    @Override
+    public void playSound(SoundType sound, Vector3d position, double volume, double pitch) {
+        this.playSound(sound, position, volume, pitch, 0);
+    }
+
+    @Override
+    public void playSound(SoundType sound, Vector3d position, double volume, double pitch, double minVolume) {
+        this.playSoundEffect(position.getX(), position.getY(), position.getZ(), sound.getName(), (float) Math.max(minVolume, volume), (float) pitch);
+    }
+
+    @Override
+    public Optional<Entity> getEntityFromUUID(UUID uuid) {
+        if ((Object) this instanceof WorldServer) {
+            // TODO Should this be done in an override in a WorldServer mixin?
+            return Optional.fromNullable((Entity) ((WorldServer) (Object) this).getEntityFromUuid(uuid));
+        }
+        for (net.minecraft.entity.Entity entity : this.loadedEntityList) {
+            if (entity.getUniqueID().equals(uuid)) {
+                return Optional.of((Entity) entity);
+            }
+        }
+        return Optional.absent();
+    }
+
+    @Override
+    public Optional<String> getGameRule(String gameRule) {
+        if (this.worldInfo.getGameRulesInstance().hasRule(gameRule)) {
+            return Optional.of(this.worldInfo.getGameRulesInstance().getGameRuleStringValue(gameRule));
+        }
+        return Optional.absent();
+    }
+
+    @Override
+    public void setGameRule(String gameRule, String value) {
+        this.worldInfo.getGameRulesInstance().setOrCreateGameRule(gameRule, value);
+    }
+
+    @Override
+    public Map<String, String> getGameRules() {
+        GameRules gameRules = this.worldInfo.getGameRulesInstance();
+        // TODO Could expose GameRules#theGameRules via reflection to remove the
```

Comment:
Actually, there's not really much advantage of doing that so I'll remove the comment.